packages/react/src/ReactCurrentBatchConfig.js

```diff
@@ -7,12 +7,19 @@
  * @flow
  */
 
+import type {Fiber} from 'react-reconciler/src/ReactInternalTypes';
+
 /**
  * Keeps track of the current batch's configuration such as how long an update
  * should suspend for if it needs to.
  */
 const ReactCurrentBatchConfig = {
   transition: (0: number),
+  _updatedFibers: (null: Set<Fiber> | null),
```

Comment:
Nit: Since this is a dev-only field, let's add the field conditionally below so that it gets stripped out of the prod build entirely. So then the type of the config object becomes:

```js
type BatchConfig = {
  transition: number,
  _updatedFibers?: Set<Fiber>, // only exists in dev
}
```

Reply:
Hm I thought I tried that but got a flow error. Let me try it again.